spec/classes/repo_spec.rb

```diff
@@ -1,8 +1,16 @@
 require 'spec_helper'
 
-describe 'hashi_stack::repo' do
+describe 'hashi_stack::repo', type: 'class' do
```

Comment:
AFAIK this isn't needed and rspec-puppet will derive it from the file path (`spec/classes`).

Reply:
fixed in 44a46657826230706a1d05ab92f4d0c391b9f457